content/news/releases/prestashop-8-2-0-available.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title:  PrestaShop 8.2 Is Available
+subtitle: A new minor version of PrestaShop is now available
+date: 2024-09-10
```

Comment:
Comment here to change the date before publishing.

Reply:
done 👍